templates/start-nexus-iq-server.sh.erb

```diff
@@ -9,4 +9,4 @@
 %>
 
 cd <%= node['nexus_iq_server']['install_dir'] %>
-exec /usr/bin/java -jar <%= nexus_iq_jar  %> server <%= nexus_iq_config  %>
+exec /usr/bin/java -jar ${JAVA_OPTS} <%= nexus_iq_jar %> server <%= nexus_iq_config %>
```

Comment:
it just looks odd to me that JAVA_OPTS is between -jar and the .jar file. according to the docs it should be `java [ options ] -jar file.jar`

Reply:
Corrected the order.